README.md

```diff
@@ -122,9 +122,9 @@ All commands use discords slash commands (/)
 
 ```/vote_draft``` 
 
-Upon typing the command the bot will present you with three buttons: Create, Edit, Delete.
+Upon typing the command the bot will present you with three buttons: Create, Edit, Preview, and Delete. 
```

Comment:
Upon typing the command, the bot will present you with four buttons: "Create", "Edit", "Preview", and "Delete".